src/pages/updateManagerForm.js

```diff
@@ -0,0 +1,120 @@
+import Link from "next/link";
```

Comment:
@istoga with these comments in this file, it's safe to say that this PR isn't ready yet eh? 😅

Reply:
Looks like Ibimina is planning some more work off of this PR

Reply:
@msarit can this pr be merge? I want to work on the unpublished functionality

Reply:
@ibimina Do you mean Unpublished functionality, or the Approved (to be visible on the landing page) functionality?

I believe we descoped the Publish/Unpublish functionality; have we decided to pick it up again?

Reply:
I meant the delete/unpublish community

Reply:
> 

Hey @ibimina I believe Sheyi wrote a ticket for the delete part. She wants to work on functionality, but I'll assign my unpublish ticket over to you.